src/main/java/be/syntra/devshop/DevshopBack/factories/OrderContentFactory.java

```diff
@@ -0,0 +1,31 @@
+package be.syntra.devshop.DevshopBack.factories;
+
+import be.syntra.devshop.DevshopBack.entities.OrderContent;
+import be.syntra.devshop.DevshopBack.entities.Product;
+import be.syntra.devshop.DevshopBack.repositories.ProductRepository;
+import lombok.NoArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.Random;
+
+@Component
+@NoArgsConstructor
+public class OrderContentFactory {
+    private ProductRepository productRepository;
+
+    @Autowired
+    public OrderContentFactory(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    public OrderContent of() {
+        List<Product> productList = productRepository.findAll();
+        int productListLength = productList.size();
```

Comment:
technically: productListLength is used only once. So you could inline this. On the other hand, new Random() is used twice and could be argued to be extracted to a method variable :)

Reply:
inlined it and extracted a getNextRandomInt(..) method